app/score_calculator/divide/thirteen_orphans_shape.py

```diff
@@ -0,0 +1,16 @@
+from typing import Final
+
+from app.score_calculator.enums.enums import Tile
+from app.score_calculator.hand.hand import Hand
+
+FULLY_HAND_SIZE: Final[int] = 14
+
+
+def can_divide_thirteen_orphans_shape(hand: Hand) -> bool:
+    if len(hand.call_blocks) or sum(hand.tiles) != FULLY_HAND_SIZE:
+        return False
+
+    for tile in range(Tile.M1, Tile.F0):
+        if Tile(tile).is_outside and hand.tiles[tile] < 1:
+            return False
+    return True
```

Comment:
all이나 any 이용하는게 좋겠네요
```suggestion
    return all(condition)
    or
    return not any(condition)
```

Tile의  range를 쓰는 일이 많으면 미리 정의 하는게 의미상 더 읽히기 쉽겠네요.
range(Tile.M1, Tile.F0)이 모든 tile들을 iterate하는지 눈에 잘 안들어옵니다.
```
return all(hand.tiles[tile]>=1 for tile in OUTSIDE_TILES)
```
특히 Tile들을 iterate하는 작업이 많을 것으로 예상되는데 특정의미를 가지는 경우라면 위와같이 따로 하는것도 괜찮습니다만, 이경우에는 enum갯수가 엄청 많지는 않으니 그냥 condition에 추가하는 것도 좋겠습니다.

Reply:
@classmethod로 range에 해당하는 값을 반환하도록 바꿨습니다